.gitignore

```diff
@@ -1,10 +1,17 @@
 .DS_Store
 /.idea/
+/.vscode/
 
 .env*
 
 /site-content/site.yaml
 /site-content/build/
 
 /site-ui/node_modules/
-/site-ui/public/
\ No newline at end of file
+/site-ui/public/
+/content/_/
+/content/assets/
+/content/doc/
+/content/index.html
+/content/search-index.js
+/content/404.html
```

Comment:
adding these `/content/` lines broke the website, because we expect to commit all the generated content on the asf-staging and asf-site branches.

removing these lines.